azerothcore/azerothcore-wotlk
# fix(DB/game_event): Move Brewfest start dates.

## Changes Proposed:
- Moves the start of Brewfest up to [three in the morning](https://user-images.githubusercontent.com/38561765/192158851-f33f7777-cef5-4eb3-ad70-364392ea2e46.png). (Image taken from Wrath Classic)
- Moves the game event of building up the brewfest event in Ironforge three days earlier.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/4097
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13137
- Related to https://github.com/azerothcore/azerothcore-wotlk/issues/653

## SOURCE:

Discussions.

## Tests Performed:

- Untested.

## How to Test the Changes:


1. Run the query.
2. Time travel to September 17th.
3. Time travel to September 20th.
4. Observe events happening back to back.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
